src/strategies/base.ts

```diff
@@ -333,51 +333,96 @@ export abstract class BaseStrategy implements Strategy {
     };
   }
 
-  protected extraFileUpdates(
+  // Helper to convert extra files with globs to the file paths to add
+  private async extraFilePaths(extraFile: ExtraFile): Promise<string[]> {
+    if (typeof extraFile !== 'object') {
+      return [extraFile];
+    }
+
+    if (!extraFile.glob) {
+      return [extraFile.path];
+    }
+
+    if (extraFile.path.startsWith('/')) {
+      // glob is relative to root, strip the leading `/` for glob matching
```

Comment:
I don't understand why the glob doesn't work with the leading slash ...

Reply:
We could perhaps fix this in the upstream library, but the filenames coming from GitHub do not have a leading `/` (so the leading slash on the glob does not match)

Reply:
I see!